runtime/common/src/lib.rs

```diff
@@ -53,6 +53,12 @@ parameter_types! {
 	pub const MaximumBlockWeight: Weight = 2_000_000_000_000;
 	pub const AvailableBlockRatio: Perbill = Perbill::from_percent(75);
 	pub const MaximumBlockLength: u32 = 5 * 1024 * 1024;
-	pub const ExtrinsicBaseWeight: Weight = 100_000_000; // TODO: Confirm/Update
-	pub const BlockExecutionWeight: Weight = 1_000_000_000; // TODO: Confirm/Update
+	/// Executing 10,000 System remarks (no-op) txs takes ~1.26 seconds -> ~125 µs per tx
+	pub const ExtrinsicBaseWeight: Weight = 125_000_000;
```

Comment:
My point is, we have done some benchmark here and we have realised that with 125 µs is the base time of each extrinsic. Except for when someone has a fork of FRAME in which everything is radically different, there is no point in even having the option to configure these stuff. They should be defined somewhere in `frame_system::weight_constants` and put here so we prevent this footgun in the future.

Reply:
They should still be configurable, but the default values come from system or _somewhere_ within FRAME.

Reply:
You are probably right in the context of `ExtrinsicBaseWeight` and `BlockExecutionWeight`. Obviously we want to allow people to modify `MaximumBlockWeight` and `DbWeight` on their own, but probably we should expose different variations of `DbWeight` from frame for that customization.

Reply:
I think we want people to change all of them, but they can only change them once they have either their custom 

1. extrinsic weight, i.e. a fork of `pallet-balances` instead of the original ones. 
2. use their own db
3. have their own block time. 

none of which happens by default and most people do not want to deal with. 

now imagine a week from now we decide to multiply all weights bu 10 again. These hardcoded numbers will all break again. Whereas if we put them somewhere within frame and circulate them inwards to the runtime again, we prevent this.

Reply:
@kianenigma I think we are in general agreement, but do note, that the solution (with regard to things like changing the balances pallet) should come from our Currency trait providing weights that we use rather than looking at the weight of a function without considering which Currency trait we are using. This is obviously a pretty big effort.

Reply:
fixed!